NSS-Day-Cohort-42/rare-media-moguls
# Ap post tags

#### Changes Made
1. Added post tags component to PostDetail.js
2. Built component PostTags to fetch tags associated with posts in db and hold button to display CurrentPostTags.js or EditPostTags.js
3. Built component to generate CurrentPostTags.js from tags associated with post
4. Built EditPostTags.js to: Fetch all available tags and display checkboxes, check all associated tags, add unchecked tags when checked, and remove checked tags when unchecked, then to re-render associated tags on exit of editor
  

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout ap-post-tags
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user selects a post, a list of associated tags (if any) should appear at the bottom of the page
> When a user clicks "manage tags" they should see a checkbox list of all tags with current tags checked
> On click, checkboxes should respond appropriately
> On click of "manage tags" again, checked tags should display and unchecked tags should be removed
